docs/src/extend/custom-rule-tutorial.md

```diff
@@ -431,9 +442,11 @@ Now you're ready to test the custom rule.
 
 Run ESLint on the `example.js` file you created in step 8, now with the downloaded plugin:
 
-```shell
-npx eslint example.js
-```
+{{ npx_tabs({
+    package: "eslint",
+    args: ["example.js"],
+    comment: null
```

Comment:
do we need these explicit `comment: null`? I see that you have added validations for this?

Reply:
yes, without specifying `comment` property, it throws an error